python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -243,14 +248,18 @@ def _getDefaultValueInternal(self, attr, val=None, check=False):
                 defvalue = _found_attrs[stored_attr_key]
             else:
                 defvalue = config[attr]
-                _found_attrs[stored_attr_key] = defvalue
+                from Ganga.GPIDev.Base.Proxy import stripProxy
+                ## Just in case a developer puts the proxied object into the default value!
+                _found_attrs[stored_attr_key] = stripProxy(defvalue)
```

Comment:
Yes I agree but I was hoping to do this in a future PR vs adding that into this. I can revert it but this may add problems which this is looking to avoid